discord/embeds.py

```diff
@@ -228,9 +228,15 @@ def from_dict(cls, data: Mapping[str, Any]) -> Self:
         # try to fill in the more rich fields
 
         try:
-            self._colour = Colour(value=data['color'])
+            color = data['color']
         except KeyError:
-            pass
+            try:
+                color = data['colour']
+            except KeyError:
+                color = None
```

Comment:
```suggestion
            color = data.get('colour', None)
```

Reply:
Unsure about that since it's either a personal preference or better practice according to Danny to use `try-except`. So going to wait on Danny for that.

Reply:
There's no extra side effects being introduced by using `.get` here (e.g. no object being over-eagerly created, no necessity for differentiating between the key being present or not), so I doubt using it here would be problematic